src/app/tailwind-templates/page.tsx

```diff
@@ -1,17 +1,28 @@
+import Navbar from "../components/Navbar"
+import Heading from "../components/Heading"
+import Subheading from "../components/Subheading"
+import P from "../components/P"
+import GreyBtn from "../components/Buttons/GreyBtn"
+import RedBtn from "../components/Buttons/RedBtn"
+import BlueBtn from "../components/Buttons/BlueBtn"
+
 export default function TailwindTemplates() {
     return (
-        <div>
-            <a href="./"><h1 className="font-semibold text-7xl m-2 p-2 text-center text-awhite">Hello from new-page</h1></a>
-            <h2 className="font-medium text-5xl m-2 p-2 text-center text-awhite">This is the sub heading...</h2>
-            <p className="w-2/4 font-medium text-lg m-auto p-2 text-center text-awhite">Lorem ipsum dolor sit amet consectetur adipisicing elit. Aut, harum dolorum laborum culpa quis commodi animi sit
-                inventore ex, doloremque nobis itaque eveniet fugiat explicabo? Eligendi sunt obcaecati dicta adipisci? Lorem ipsum
-                dolor sit amet consectetur adipisicing elit. Necessitatibus soluta, ut adipisci dignissimos iusto assumenda est
-                placeat porro quo fugit magnam molestiae nulla suscipit laborum ducimus laboriosam dolores. Illo, temporibus.</p>
-            <div className="flex m-5">
-                <button className="rounded-none bg-adarkgrey m-auto p-3 font-medium text-md text-awhite justify-center">Save Changes</button>
-                <button className="rounded-none bg-ared m-auto p-3 font-medium text-md text-awhite justify-center">Save Changes</button>
-                <button className="rounded-none bg-ablue m-auto p-3 font-medium text-md text-awhite justify-center">Save Changes</button>
+        <>
+
+            <Navbar />
+            <Heading heading="Hello from new-page" />
+            <Subheading subheading="This is the sub heading..." />
+            <P p="Lorem ipsum dolor sit amet consectetur adipisicing elit. Aut, harum dolorum laborum culpa quis commodi animi sit
+                    inventore ex, doloremque nobis itaque eveniet fugiat explicabo? Eligendi sunt obcaecati dicta adipisci? Lorem ipsum
+                    dolor sit amet consectetur adipisicing elit. Necessitatibus soluta, ut adipisci dignissimos iusto assumenda est
+                    placeat porro quo fugit magnam molestiae nulla suscipit laborum ducimus laboriosam dolores. Illo, temporibus." />
+            <div className="flex m-5 justify center">
+                <GreyBtn text="Save Changes" />
+                <RedBtn text="Save Changes" />
+                <BlueBtn text="Save Changes" />
```

Comment:
If we wanted for something to happen when the client clicks the button, how would we go around implementing that?